tests/tests/cpld_control_c48

```diff
@@ -106,6 +106,8 @@ export CHLCLIM=''
 export FRUNOFF=''
 export MOM6_RIVER_RUNOFF=False
 export MOM6_RESTART_SETTING=r
+export MOM6_DIAG_COORD_DEF_Z_FILE=oceanda_zgrid_25L.nc
```

Comment:
@JessicaMeixner-NOAA Does this MOM6_DIAG_COORD_DEF_Z_FILE need to be added to the cpld_warmstart_c48 and cpld_restart_c48? Those tests are currently commented out (I need to recreate the warmstart files that they use). Will we need this variable set for those two tests?

Reply:
@DeniseWorthen good catch, yes you will want to set that for the other C48 tests.